conf/ocsci/provider_mode_ibm_cloud_baremetal_kubevirt_clients.yaml

```diff
@@ -0,0 +1,20 @@
+---
+DEPLOYMENT:
+  cnv_deployment: true
+  metallb_operator: true
+ENV_DATA:
+  platform: "hci_baremetal"
+  cluster_type: "provider"
+  deploy_acm_hub_cluster: true
+  skip_ocs_deployment: true
+  acm_version: "2.9" # this is an example, please provide the desired version
+  acm_hub_channel: "release-2.9" # this is an example, please provide the desired version
+  hcp_version: "4.14" # this is an example, please provide the desired version
+  hosted_odf_version: "latest-stable-4.14.5" # this is an example, please provide the desired version
+  metallb_version: "4.14" # this is an example, please provide the desired version
+  cpu_cores_per_hosted_cluster: 6 # minimal, tested value for the kubevirt cluster is 6
+  memory_per_hosted_cluster: "12Gi" # minimal, tested value for the kubevirt cluster is 12Gi
+  cluster_names:
+    - "hcp414-bm2-h" # this is an example, please provide the desired cluster_names
+  cluster_paths:
+    - "~/clusters/hcp414-bm2-h/openshift-cluster-dir/auth" # this is an example, provide the desired cluster_paths
```

Comment:
It is possible. I will redo this.

Reply:
done. 
I'll tell right away that I think that it would be the best to store all related data for hosted `cluster` to the dict you proposed. 

For example: 
clusters:
  "cluster1": 
    "platform": "kubevirt"
    "cluster_path": "~/clusters/cluster1/openshift-cluster-dir"
    "cpu_cores": 6
    "memory": "12Gib"
    "odf_version": "4.16-latest"
  "cluster2": 
    "platform": "agent"
    "cluster_path": "~/clusters/cluster2/openshift-cluster-dir"
    "cpu_cores": 8
    "memory": "16Gib"
    "odf_version": "4.16.0-56"

In future it will return dividends, since we will be able to install hosted clusters with different multiple platforms, different ODF versions and different specs per cluster during one jenkins job.

I will leave this as an enhancement - https://github.com/red-hat-storage/ocs-ci/issues/9594